service/agent/server.go

```diff
@@ -69,17 +68,15 @@ func Start(ctx context.Context, app dom.AppInfo, conf *Config) error {
 	logger.Info().Msg("app redis connected")
 	verSvc := meta.NewVersionService(appRedis)
 
-	confRedis := redis.NewClient(&redis.Options{Addr: conf.Redis.Address, Password: conf.Redis.Password, DB: conf.Redis.ConfigDB})
-	logger.Info().Interface("redis_pool", appRedis.PoolStats()).Int("redis_pool_size", appRedis.Options().PoolSize).Msg("redis conf pool stats")
+	confRedis := util.NewRedis(conf.Redis, conf.Redis.ConfigDB)
 	defer confRedis.Close()
 	err = redisotel.InstrumentTracing(confRedis, redisotel.WithTracerProvider(tp))
 	if err != nil {
 		return fmt.Errorf("%w: unable to instrument tracing app redis", err)
 	}
 	policySvc := policy.NewService(confRedis)
 
-	queueRedis := asynq.RedisClientOpt{Addr: conf.Redis.Address, Password: conf.Redis.Password, DB: conf.Redis.QueueDB}
-	logger.Info().Int("redis_pool_size", queueRedis.PoolSize).Msg("redis queue pool stats")
+	queueRedis := util.NewRedisAsynq(conf.Redis, conf.Redis.QueueDB)
```

Comment:
It is possible to construct asynq client from `UniversalClient`, so, probably, there is no sense to constuct 2 configs.

See https://github.com/hibiken/asynq/blob/v0.25.0/client.go#L46

Reply:
Fair enough